src/stores/WalletStore.ts

```diff
@@ -66,6 +136,14 @@ class WalletStore extends PersistableStore {
     }
   }
 
+  connectOrChangeNetwork(
```

Comment:
It seems that by calling `connectOrChangeNetwork` without any parameters, the network should change if necessary by default

Reply:
oh, that's sound logical, I'll change it to `changeNetworkOrConnect`

Reply:
maybe it's better to replace `clearCachedProvider = false` with `useCachedProvider = true`?

Reply:
I renamed, but I dont think that we need to rename it